chain/jsonrpc-primitives/src/types/maintenance.rs

```diff
@@ -1,16 +1,17 @@
 use serde_json::Value;
 
-pub type RpcMaintenanceWindowsResponse =
-    Vec<(near_primitives::types::BlockHeight, near_primitives::types::BlockHeight)>;
+pub type RpcMaintenanceWindowsResponse = Vec<std::ops::Range<near_primitives::types::BlockHeight>>;
```

Comment:
@saketh-are Do you know if anyone relies on the RpcMaintenanceWindowResponse to be in the following format:

```
[[100500, 100600], [100700, 100800]]
```

This change proposes to make the output more clear (and JSON Schema will also be more sane):

```
[{"start": 100500, "end": 100600], {"start": 100700, "end": 100800}]
```

Reply:
I have no idea really. Considering the apparent purpose of the query it seems reasonably safe to change. It doesn't seem like anything that should have significant automation built on top of it.